packit_service/worker/reporting.py

```diff
@@ -125,14 +146,24 @@ def set_status(
             self.project.set_commit_status(
                 self.commit_sha, state, url, description, check_name, trim=True
             )
-        except gitlab.exceptions.GitlabCreateError:
+        except gitlab.exceptions.GitlabCreateError as e:
             # Ignoring Gitlab 'enqueue' error
             # https://github.com/packit-service/packit-service/issues/741
-            pass
+            if e.response_code == 403:
+                # In case we don't have permissions to set status we post comment
+                self.__add_commit_comment(state, description, check_name, url)
```

Comment:
Maybe something about status in the name?
```suggestion
                self.__add_commit_comment_with_status(state, description, check_name, url)
```

or

```suggestion
                self.__put_status_to_commit_comment(state, description, check_name, url)
```